components/create_comment/create_comment.jsx

```diff
@@ -341,6 +338,13 @@ class CreateComment extends React.PureComponent {
         }
     }
 
+    saveDraft = () => {
+        if (this.saveDraftFrame) {
```

Comment:
I know we didn't do this before, but maybe we should clear `this.saveDraftFrame` in this block to prevent clearing it and attempting to save the draft multiple times?